src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/todo';
```

Comment:
add this file to git

Reply:
@TarasHoliuk not sure what I should do herE:
![image](https://user-images.githubusercontent.com/101051619/233640916-77c1b591-b9a1-4fd5-bb25-c37b1c32d265.png)

Reply:
didn't fix it, I don' see folder types 
<img width="340" alt="Screenshot 2023-04-21 at 16 26 51" src="https://user-images.githubusercontent.com/53228259/233647716-c553cc24-c008-4b23-bf77-24d8fc291c0b.png">